docs/pages/includes/database-access/reference/aws-iam/elasticache/access-policy.mdx

```diff
@@ -1,6 +1,41 @@
-{/* this comment is here to make the include below it render */}
+ElastiCache supports IAM authentication for Redis engine version
+7.0 or above.
+This is the recommended way to configure Teleport access to ElastiCache.
 
-(!docs/pages/includes/database-access/reference/aws-iam/redis/access-policy.mdx dbType="ElastiCache" permissionType="elasticache"!)
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ElastiCacheFetchMetadata",
+            "Effect": "Allow",
+            "Action": "elasticache:DescribeReplicationGroups",
+            "Resource": "*"
+        },
+        {
+            "Sid": "ElastiCacheDescribeUsers",
+            "Effect": "Allow",
+            "Action": "elasticache:DescribeUsers",
+            "Resource": "*"
+        },
+        {
+            "Sid": "ElastiCacheConnect",
+            "Effect": "Allow",
+            "Action": "elasticache:Connect",
+            "Resource": [
+                "arn:aws:elasticache:<Var name="us-east-2"/>:<Var name="aws-account-id"/>:replicationgroup:<Var name="replication-group"/>",
```

Comment:
I'd add instructions to assign a value to this variable above the snippet to avoid the Vale warning